samples/react-birthdays/package.json

```diff
@@ -1,58 +1,47 @@
 {
-  "name": "birthdays-anniversary",
-  "version": "2.0.0",
+  "name": "happy-birthday",
+  "version": "0.0.1",
```

Comment:
we should  either not change the version number to make it higher but not smaller